config-default.yml

```diff
@@ -512,7 +503,6 @@ duck_pond:
         - *PYNEWS_CHANNEL
         - *PYEVENTS_CHANNEL
         - *MAILING_LISTS
-        - *REDDIT_CHANNEL
```

Comment:
Does removing this make sense? The channel will continue to exist, right? This is config for a different feature.

Reply:
we could keep them 👍